contrib/publish-arm.sh

```diff
@@ -2,6 +2,7 @@
 
 declare -a repos=("openfaas-incubator/openfaas-operator" "openfaas-incubator/faas-idler" "openfaas/faas" "openfaas/faas-swarm" "openfaas/nats-queue-worker" "openfaas/faas-netes" "openfaas/faas-cli")
 HERE=`pwd`
+ARCH=$(uname -m)
```

Comment:
Was this tested on both variants?

Reply:
Yes, I believe. I'll give it a go these days once again though just to be sure 👍